doc/modules/mixture.rst

```diff
@@ -135,6 +135,38 @@ parameters to maximize the likelihood of the data given those
 assignments. Repeating this process is guaranteed to always converge
 to a local optimum.
 
+Choice of Initialization Method
+-------------------------------
+
+There is a choice of four initialization methods (as well as inputting user defined
+initial means) to generate the initial centers for the model components. The default
+is to apply a traditional kmeans clustering algorithm. This can be computationally
+expensive so other initialization options provided are
+
+* *k-means++* - This uses the initialization method of kmeans clustering. k-means++
```

Comment:
Note that the right way to do definition lists in in restructured text is:

```
k-means++
    This uses the initialisation method of kmeans clustering. k-means++
    will...
rand_data
    This will pick random data points
random
    Centers are chosen
```